apps/recovery-utility/renderer/components/RecoveryForm/index.tsx

```diff
@@ -174,15 +175,22 @@ export const RecoveryForm = ({ verifyOnly }: Props) => {
                 {...register('agpRsaPassphrase')}
               />
             ) : (
-              <TextField
-                id='passphrase'
-                type='password'
-                label='Mobile App Recovery Passphrase'
-                helpText='Set by the workspace owner during onboarding'
-                error={errors.passphrase?.message}
-                disabled={recoverMutation.isLoading}
-                {...register('passphrase')}
-              />
+              <Tooltip title={recoverOnlyNCW ? 'Mobile passphrase is not required for NCW wallets recovery' : ''}>
+                <TextField
+                  id='passphrase'
+                  type={recoverOnlyNCW ? 'text' : 'password'}
+                  label='Mobile App Recovery Passphrase'
+                  helpText='Set by the workspace owner during onboarding'
+                  error={errors.passphrase?.message}
+                  disabled={recoverMutation.isLoading || recoverOnlyNCW}
+                  {...register('passphrase')}
+                  sx={{
+                    '& input[disabled]': {
+                      backgroundColor: 'rgba(0, 0, 0, 0.12)',
+                    },
+                  }}
```

Comment:
why was this added?

Reply:
to change the color of the input field when disabled to grey